### ISSUE TITLE
Allow CS use to refill NP

### ISSUE THREAD

**Issue by OfficerArtoria** [Fate-Grand-Automata/FGA#1869]
### Preparation

- [X] No one has asked for this feature before in other issues (even the closed ones)

### Describe the feature

In light of the most recent update on the Japanese server, there are scenarios where using Command Spells have become feasible, specifically for Bleached Earth Free Quests. Since 3 CS regen per day, and each day players are provided with 3 Storm Pods, players can now reasonably equip BG on their DPS, use the CS to charge NP.

**Comment by tkp206093** [comment]
Yes, Recommended that can set up Command Spell for use in scripts. The Japanese server and the US server have also been updated with three Command Spell per day. You can use Command Spell to make clean the ordeal call.

**Comment by BalletDuckNinja** [comment]
Just want to add support for this feature, it would make daily bleached earth farming much more flexible so you can bond farm with more servants of choice

**Comment by Djinnistorm** [comment]
Is this still a planned feature/being worked on, or is this abandoned? It would be nice just to save some repetitive presses every day while opening up team comps to bond with. I'm sure at least a few people would appreciate this feature, but if it's not in the cards I understand.
